lib/vagrant-service-manager/plugin.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'providers/docker'
```

Comment:
+1 Service is the better term. I think we need to aim for a consistent terminology not only in this plugin, but across the whole ADB/CDK. "The CDK offers a OpenShift service" or "To enable the Kubernetes service ...", etc